packages/common/src/types/coder.ts

```diff
@@ -41,22 +44,22 @@ export type ApiChartAsCodeUpsertResponse = {
     results: PromotionChanges;
 };
 
-export type DashboardTileWithoutUuids = Omit<
-    DashboardTile,
-    'properties' | 'uuid'
-> & {
+export type DashboardTileAsCode = Omit<DashboardTile, 'properties' | 'uuid'> & {
     uuid: DashboardTile['uuid'] | undefined; // Allows us to remove the uuid from the object
-    properties: Omit<
-        DashboardTile['properties'],
-        'savedChartUuid' | 'savedSqlUuid' | 'savedSemanticViewerChartUuid'
-    >;
+    properties:
```

Comment:
include markdown and loom properties in here, so the type is accurate with all the supported tiles